core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -110,9 +112,24 @@ abstract contract FeePayer is Testable {
         );
         lastPaymentTime = time;
 
+        totalPaid = regularFee.add(latePenalty);
+        if (totalPaid.isEqual(0)) {
+            return totalPaid;
+        }
+        // If the effective fees paid as a % of the pfc is > 100%, then reduce it to 100%. The amount that `totalPaid` is greater than `_pfc` is
+        // reduced from the greater component of (regularFees, latePenalty).
+        if (totalPaid.isGreaterThan(_pfc)) {
+            if (regularFee.isGreaterThan(latePenalty)) {
+                regularFee = regularFee.sub((totalPaid.sub(_pfc)));
+            } else {
+                latePenalty = latePenalty.sub((totalPaid.sub(_pfc)));
+            }
+
+            totalPaid = regularFee.add(latePenalty);
+        }
+
```

Comment:
One kinda weird outcome of doing this is that the cumulative multiplier goes to 0. This means that everyone's position has 0 collateral (which is expected). But it also means that the contract is basically dead -- if anyone tries to enter the contract afterwards, there will be a divide by 0 error when trying to add collateral.

Reply:
Yeah exactly. There's a tradeoff of either reverting before the cumulative multiplier gets to 0 or allowing it which will kill the contract from practical usage

Reply:
This will also cause all withdrawals/settleExpired's to fail

Reply:
Yeah, I'd expect that every method would either revert or do nothing.